src/components/header/ThreadMarkerOverlay.js

```diff
@@ -2,17 +2,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { PureComponent, PropTypes } from 'react';
+// @flow
 
+import React, { PureComponent, PropTypes } from 'react';
+import type { Thread } from '../../types/profile';
 import './ThreadMarkerOverlay.css';
 
+type Props = {
+  rangeStart: number,
+  rangeEnd: number,
+  thread: Thread,
+  onSelectMarker: number => *,
+};
```

Comment:
This file is actually not used anywhere, I think... so you can remove the proptypes here as we'll take care than any new component that would use this component will be flow-checked.